Code/Framework/AzQtComponents/AzQtComponents/Components/FilteredSearchWidget.cpp

```diff
@@ -232,6 +234,43 @@ namespace AzQtComponents
 
         QMenu::showEvent(e);
     }
+    bool SearchTypeSelector::eventFilter(QObject* obj, QEvent* event)
```

Comment:
uber nit: Can we add a line break between function calls please

Reply:
Done